### ISSUE TITLE
Step 3 | Income

### ISSUE THREAD

**Issue by jfoclpf** [jfoclpf/autocosts#36]
On the "What's your net income?" currently the user has the option to choose per: year, month, week and hour, and then base on the selection the user gets different fields. I was wondering if it would be possible to make it simpler, like this:

![image](https://user-images.githubusercontent.com/3984909/40586233-f727b89a-61bf-11e8-9cab-43e7dd20fe95.png)

**Comment by jfoclpf** [comment]
@mcristaldo 

I don't clearly understand the question, yes, in principle it should be made like that, exactly as it is already

![image](https://user-images.githubusercontent.com/3984909/40746489-dca0e47c-645a-11e8-8b21-d779b34a44de.png)

but then, according to the user input, other fields should appear, because if you insert the income per month, you need to know how many months per year the user worked

if the user inserts per hour, we need to know how much he earns per hour, but how many hours per week and so on.
![image](https://user-images.githubusercontent.com/3984909/40746628-3839d910-645b-11e8-9809-3d3408fa0939.png)

Basically we need to know the _yearly income_ of the user with different methods.

Therefore all those fields are indeed necessary. 

Keep the `ids` of all those fields and everything should be run (hide and show) ok.

**Comment by jfoclpf** [comment]
These are the functions that deal with those "hides and shows". If you keep the ids (#) and structure, everything should render ok

```js
function income_toggle(value){
    switch(value){
        case "year":
            $("#income_per_year_form3, #working_time_form3").show("slow");
            $("#income_per_month_form3, #income_per_week_form3, #income_per_hour_form3").hide("slow");
            income="year";
            break;
        case "month":
            $("#income_per_month_form3, #working_time_form3").show("slow");
            $("#income_per_year_form3, #income_per_week_form3, #income_per_hour_form3").hide("slow");
            income="month";
            break;
        case "week":
            $("#income_per_week_form3, #working_time_form3").show("slow");
            $("#income_per_year_form3, #income_per_month_form3, #income_per_hour_form3").hide("slow");
            income="week";
            break;
        case "hour":
            $("#income_per_hour_form3").show("slow");
            $("#income_per_year_form3, #income_per_week_form3, #income_per_month_form3, #working_time_form3").hide("slow");
            income="hour";
            break;
    }
}

//radio button toggle function of "Working time" section in form part 3
function working_time_toggle(value){
    if(value){
        //selects actively radio button to which this function is associated
        $("#working_time_yes_form3").prop("checked", true);
        $("#working_time_input_form3").show("slow");
    }
    else{
        //selects actively radio button to which this function is associated
        $("#working_time_no_form3").prop("checked", true);
        $("#working_time_input_form3").hide("slow");
    }
}
```

**Comment by mcristaldo** [comment]
Done
It has been updated to be the same as the current site. 
`https://github.com/jfoclpf/autocosts/tree/a1bf638a627018cd51ef58982a85638b7cf64cc7`
